src/icons/fileIcons.ts

```diff
@@ -206,6 +206,7 @@ export const fileIcons: FileIcons = {
         { name: 'cpp', fileExtensions: ['cc', 'cpp', 'mm', 'cxx'] },
         { name: 'hpp', fileExtensions: ['hpp'] },
         { name: 'go', fileExtensions: ['go'] },
+        { name: 'go-mod', fileExtensions: ['go.mod', 'go.sum'] },
```

Comment:
It would be better if you could rename the icon to `go_mod`, so just replace the hyphen with an underscore.